src/diagnostics/core_diagnostics.jl

```diff
@@ -916,15 +894,13 @@ function compute_clvi!(
             ).z
         Operators.column_integral_definite!(
             out,
-            Base.broadcasted(
-                *,
+            LB.@lazy_broadcasted @. (
```

Comment:
In an offline discussion (https://github.com/CliMA/ClimaCore.jl/pull/1947#issuecomment-2303361253), we decided that we will keep broadcasted stuff out of Atmos until there's a clear pathway to learn about it and packages are properly documented, e.g. https://github.com/CliMA/LazyBroadcast.jl/issues/7).

So, you can just pass a `Field`. To reduce alllcations, you can use one of the scratch fields in the cache.